benchmarks/fortune/fortune.py

```
@@ -0,0 +1,123 @@
+                    current_neg.append(line[1:])
+                elif line[0] == '-':
+                    current_pos.append(line[1:])
+                elif line[0] == '@':
+                    problems.append(('fail', "`%s` should be `%s`" %
+                                     (''.join(current_neg),
+                                      ''.join(current_pos)), url))
+            if len(current_pos) != 0:
+                problems.append(('fail', "`%s` should be `%s`" %
+                                 (''.join(current_neg),
+                                  ''.join(current_pos)), url))
+        except:
```

Ruff

Do not use bare `except`

(E722)